tests/fixtures/templates/example4.yml

```diff
@@ -31,6 +37,8 @@ Globals:
           Fn::GetAtt: [ ApiGatewayTwo, Region ]
         STAGE_NAME: 
           Fn::GetAtt: Stage.Region
+        HANDLER: 
+          Fn::Select: [ !FindInMap [Environments, !Ref Environment, Index], ["handler1", !Ref Handler]]
```

Comment:
Acho que esse exemplo é inválido, você só pode ter funções na lista de valores, nunca no índice.

Reply:
O index aceita essas duas funções:

For the Fn::Select index value, you can use the Ref and Fn::FindInMap functions.